### PR TITLE
Add minimal Sphinx for ReadTheDocs

### PR DESCRIPTION
**Pull request by nottombrown**
image

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.md'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'CleverHans'
+copyright = '2017, Ian Goodfellow, Nicolas Papernot, Ryan Sheatsley'
```
Comment (goodfeli) [inline docs/conf.py]: Can we delete this line?

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+author = 'Ian Goodfellow, Nicolas Papernot, Ryan Sheatsley'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '2.0.0'
```
Comment (goodfeli) [inline docs/conf.py]: Can we get this to say '-dev' when it's not run directly on the tagged release?

File: docs/conf.py (hunk 1 above)
Comment (goodfeli) [reply docs/conf.py]: (the line about "copyright")

**Comment by goodfeli** [comment]
Made two small comments. Looks great, thanks!

**Comment by nottombrown** [comment]
Removed version number and added better introductory text. Up to date version here:
image

**Comment by goodfeli** [comment]
LGTM. Feel free to merge once the tests pass

**Pull request merged**